### ISSUE TITLE
[Dark Mode] PrintPreviewControl does not display correctly after setting the background to black

### ISSUE THREAD

**Issue by Nora-Zhou01** [dotnet/winforms#11940]
### .NET version

9.0.0-rc.1.24415.1 + build private dll from winforms repo main branch

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

This is a new feature Dark Mode: https://github.com/dotnet/winforms/pull/11857

### Issue description

Dark Mode：
PrintPreviewControl does not display correctly after setting the background to black
![print1](https://github.com/user-attachments/assets/c14fba45-20a1-4790-8f9c-12c291c5fa6c)
Classic Mode:
PrintPreviewControl can be displayed correctly after the background is set to black
![print2](https://github.com/user-attachments/assets/bb719e83-f69c-45fc-bda6-27d07aa22522)

### Steps to reproduce

1. Create a WinForms .NET Core application with PrintPreviewControl control
2. PrintPreviewControl backcolor is set to black
3. Enable DarkMode by Application.SetColorMode(SystemColorMode.Dark) in Program.cs file
4. Run the application

**Comment by Liv-Goh** [comment]
Tested in the latest .NET 10 SDK build: 10.0.100-preview.3.25126.20, the issue does not repro anymore, so close this issue. 

![Image](https://github.com/user-attachments/assets/9f562a22-9acd-41c9-b246-d73b9c88b430)
